packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/PostWithArtGrid.tsx

```diff
@@ -0,0 +1,174 @@
+      leftFlipped: true, // for the 2025+ styling (for the 2023) and onward, we want to flip
+      // the left-side images because the images are designed to have most of the content on the right side by default (but we want it to show up on the left there)
+      middleXPct: .66,
+      middleYPct: .15,
+      middleWidthPct: .33,
+      middleHeightPct: 1,
+      middleFlipped: false,
+      rightXPct: 0,
+      rightYPct: .15,
+      rightWidthPct: .33,
+      rightHeightPct: .65, 
+      rightFlipped: false,
+    } });
+
+    if (errors) {
+      // eslint-disable-next-line no-console
+      console.error('Error saving coordinates', errors);
+    } else {
+      setImageInfo(image);
+    }
+  }
+
+  useEffect(() => {
+    if (!selectedImageInfo) {
+      const newImageInfos = images.filter(image => 
+        image.activeSplashArtCoordinates && 
+        image.activeSplashArtCoordinates.reviewWinnerArtId === image?._id
+      );
+      const newImageInfo = newImageInfos.sort((a, b) => {
+        const dateA = a.activeSplashArtCoordinates?.createdAt ? new Date(a.activeSplashArtCoordinates.createdAt).getTime() : 0;
+        const dateB = b.activeSplashArtCoordinates?.createdAt ? new Date(b.activeSplashArtCoordinates.createdAt).getTime() : 0;
+        return dateB - dateA;
+      })[0];
+      if (newImageInfo) {
+        setImageInfo(newImageInfo);
+      }
+    }
+  }, [selectedImageInfo, setImageInfo, images]);
+
+  return <div>
+    {!defaultExpanded && <div className={classes.expandButton} onClick={() => setExpanded(!expanded)}>{expanded ? 'Collapse' : `Expand (${images.length})`}</div>}
+
+    {expanded && Object.entries(imagesByPrompt).map(([prompt, promptImages]) => {
+      const corePrompt = prompt.split(artPrompt)[0]
+      return <div key={prompt} className={classes.row}>
+        <h3>{corePrompt}</h3>
+        <div className={classes.content}>
+          <GenerateImagesButton 
+            postId={post._id}
+            prompt={prompt}
+            allowCustomPrompt={false}
+            buttonText="Generate More With This Prompt"
+          />
+          <div className={classes.postWrapper} >
+            {promptImages.map((image) => {
+              const smallUrl = getCloudinaryThumbnail(image.splashArtImageUrl);
+              const medUrl = getCloudinaryThumbnail(image.splashArtImageUrl, 800);
+
+              const tooltip = <img src={medUrl} className={classes.imageTooltipContainer} />
+
+              return <LWTooltip key={image._id} title={tooltip} tooltip={false}>
+                <div key={image._id} className={classes.imageWrapper}>
+                  <img className={classNames(classes.image, selectedImageInfo?._id === image._id && classes.selectedImage)} src={smallUrl} onClick={() => handleSaveCoordinates(image)} />
+                  <div className={classes.imageId}>
+                    {image._id}
+                  </div>
+                </div>
+              </LWTooltip>
+            })} 
```

Comment:
supererogatory, but this really feels like it wanted to be a separate, small component inside of this file, or something

Reply:
not gonna fix